Makefile

```diff
@@ -5,7 +5,7 @@ NAME = "github.com/goto/optimus"
 LAST_COMMIT := $(shell git rev-parse --short HEAD)
 LAST_TAG := "$(shell git rev-list --tags --max-count=1)"
 OPMS_VERSION := "$(shell git describe --tags ${LAST_TAG})-next"
-PROTON_COMMIT := "39899a3ef41da1d15590ebada57d5e1dbce39e2d"
+PROTON_COMMIT := "b390e2b91f6cb645803317075ab2082fcfaabdd2"
```

Comment:
let's do `make generate-proto`, as there will be some changes from other commits